OpenRA.Mods.Cnc/FileFormats/LZOCompression.cs

```diff
@@ -250,7 +250,11 @@ static unsafe int LZO1xDecompress(byte* @in, uint inLen, byte* @out, ref uint ou
 					{
 						*op++ = *ip++;
 						if (t > 2)
+						{
+							#pragma warning disable IDE0047
 							(*op++) = *ip++;
+							#pragma warning restore IDE0047
```

Comment:
Why do we need to disable this here? Are the parentheses not unnecessary but the warning still appears? Line 251 has the same? 🤔

Reply:
Try removing parenthesis. Something very weird is going on

Reply:
If you remove the parenthesis rule SA1023 kicks in:
> Dereference symbol '*' should not be preceded by a space. csharp(SA1023)